monitoring_service.rst

```diff
@@ -4,36 +4,24 @@ Monitoring Service
 ##################
 
 
-Summary
-=======
-*   A Monitoring Service (MS) listens to Monitor Requests (MRs - blinded Balance Proofs) in a public "broadcast" Matrix room. An MR is accompanied by an offered reward for acting on it.
-*   Any pre-registered MS can decide to monitor a channel and store the corresponding MRs.
-*   Then, whenever a channel is closed by calling ``closeChannel`` and if the client did not react within a specified
-    timeout, the MS will call ``updateNonClosingBalanceProof`` with the submitted MR on behalf of its client.
-*   Rewards are paid in RDN, there is no free tier.
-*   In the current stage, this is a simple design which is expected to help to reach the Ithaca milestone earlier
-*   In the long-term a PISA approach seems more economically viable and user friendly, but this design requires
-    additional features and can be developed independently after Ithaca.
-
-Usual Scenario
-==============
+Overview
+========
+
+Monitoring services watch open payment channels when the user is not online. In
+case one channel partner closes a channel while the counterparty is offline (or
+doesn’t react for 80% of the settlement timeout after close has been called),
```

Comment:
The 80% is not the whole truth. We have a time range when it reacts and use an approximation for the settlement period. We could at least add a `~` to not trick anyone into thinking this is precise.